SampleContrastClient/Program.cs

```diff
@@ -86,7 +86,7 @@ static void Main(string[] args)
 
                         //foreach (Trace trace in traces)
                         //{
-                        //    Console.WriteLine("Trace Exists:{0}", DoesTraceExistForUrl(client, appId, trace.Request.Uri));
```

Comment:
Just curious: why are we updating commented code?

Reply:
I think it was meant to work as example hence why I updated.